src/TinyCiv.Server/Handlers/UnitAddHandler.cs

```diff
@@ -11,7 +12,7 @@ public class UnitAddHandler : ClientHandler<CreateUnitClientEvent>
 {
     private readonly IGameService _gameService;
 
-    public UnitAddHandler(ILogger<UnitAddHandler> logger, IGameService gameService) : base(logger)
+    public UnitAddHandler(ILogger<UnitAddHandler> logger, IGameService gameService, IPublisher publisher) : base(publisher, logger)
```

Comment:
@devJUKI FYI now `IPublisher` needs to be used for notifying UI, although it is still wrapped inside those ```NotifyCallerAsync``` and ```NotifyAllAsync``` BaseHandler methods

Reply:
It can also be injected into services, however, not sure if we should do that, since we have multiple places where we pass in "callback"/"notifier" methods

Reply:
But we could refactor little by little... As in, when we are touching the part of code where "callback" is used, only then change etc